### DISCUSSION TITLE
Welcome to MACS Discussions!

### DISCUSSION THREAD

**Discussion by taoliu** [macs3-project/MACS#431] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by maxrwjones** [comment]
Hi, I've just switched from using an older version of MACS2 (v2.0.10) to a newer one (v2.2.7.1) and I notice that the `--shift-control` option no longer exists. Has this been removed or is it now built-in/default?

I'm doing ATAC-seq for the first time, for which (as far as I can see) shifting then extending the control tags in the same manner as the test tags is definitely a good idea.

Thanks for your time!
